cli.js

```diff
@@ -158,7 +158,10 @@ function exit(results) {
 	// correctly flush the output when multiple test files
 	process.stdout.write('');
```

Comment:
oops. Sorry - `blame` attributed it to you, and I didn't scan down the #47 far enough.